src/main/java/org/folio/bulkops/controller/FqlQueryController.java

```diff
@@ -0,0 +1,6 @@
+package org.folio.bulkops.controller;
+
+import org.folio.querytool.rest.resource.FqlQueryApi;
+
+public class FqlQueryController implements FqlQueryApi {
```

Comment:
Please, create stubs for FqlQueryApi interface.

Reply:
done